libraries/AP_HAL_ChibiOS/hwdef/iomcu-f103-8Mhz-dshot/hwdef.inc

```diff
@@ -0,0 +1,98 @@
+# hw definition file for processing by chibios_pins.py
+
+include ../iomcu/hwdef.inc
+
+# defines to allow the loop timing to be observed with a Saleae
```

Comment:
This should include iomcu-f103-dshot/hwdef.inc and almost all the lines should be deleted

Reply:
Nothing actually includes this so I don't think there is a need for another hwdef.inc - just use hwdef.dat

Reply:
You are right - just use hwdef.dat in iomcu-f103-8Mhz-dshot/hwdef.dat.